contracts/voting-verifier/tests/tests.rs

```diff
@@ -1,57 +1,58 @@
 use axelar_wasm_std::voting::Vote;
 use cosmwasm_std::{from_binary, Addr, Uint64};
-use cw_multi_test::{App, ContractWrapper, Executor};
+use cw_multi_test::{App, Executor};
 
 use axelar_wasm_std::operators::Operators;
-use axelar_wasm_std::{nonempty, Threshold, VerificationStatus};
+use axelar_wasm_std::{nonempty, VerificationStatus};
 use connection_router_api::{ChainName, CrossChainId, Message, ID_SEPARATOR};
+use integration_tests::contract::Contract;
 use mock::make_mock_rewards;
 use service_registry::state::Worker;
 use voting_verifier::events::TxEventConfirmation;
-use voting_verifier::{contract, error::ContractError, msg};
+use voting_verifier::{error::ContractError, msg};
 
 use crate::mock::make_mock_service_registry;
+use crate::test_utils::VotingVerifierContract;
 
 pub mod mock;
+mod test_utils;
 
 const SENDER: &str = "sender";
 const POLL_BLOCK_EXPIRY: u64 = 100;
 fn source_chain() -> ChainName {
     "source_chain".parse().unwrap()
 }
 
-fn initialize_contract(app: &mut App, service_registry_address: nonempty::String) -> Addr {
-    let rewards_address = make_mock_rewards(app).into();
+struct TestConfig {
```

Comment:
don't call it config. Config(uration) usually means a set of parameters you use to set up the environment. Alternative suggestions: TestSetup, TestFixure, TestEnv

Reply:
when you change it make sure you change the variable name in the tests, too

Reply:
Thanks, fixed in latest commit.